src/SWP/Bundle/ContentBundle/Resources/config/serializer/CmfRoutingBundle/Doctrine.Phpcr.Route.yml

```diff
@@ -0,0 +1,11 @@
+Symfony\Cmf\Bundle\RoutingBundle\Doctrine\Phpcr\Route:
```

Comment:
why not `SWP\Bundle\ContentBundle\Document\Route`? it extends `Symfony\Cmf\Bundle\RoutingBundle\Doctrine\Phpcr\Route` - we should not depend directly on that 3rd party class here imo, you will not need to import those namespaces into jms serializer